api/files/serializers.py

```diff
@@ -14,6 +14,12 @@
 from api.base.serializers import Link, JSONAPISerializer, LinksField, IDField, TypeField
 from website.util import api_v2_url
 
+
+class SerializerIntegerField(ser.SerializerMethodField):
```

Comment:
Let's rename this to `SerializerMethodIntegerField` so it's clear where it came from and how it behaves.